Cargo.lock

```diff
@@ -2875,7 +2875,7 @@ dependencies = [
 
 [[package]]
 name = "pallet-contract-operator"
-version = "0.1.0"
+version = "1.0.0"
```

Comment:
Could you version up all plasm pallets `1.0.0` to `1.0.1`?

Reply:
Do you plan to follow all pallets to patch version of node? It looks a bit redundantly.
What’s about to follow minor version? Like “1.1.0”, “1.2.0”, etc.

Reply:
Yes, I'm convinced.